cpp/benchmarks/io/orc/orc_reader_benchmark.cpp

```diff
@@ -107,8 +106,7 @@ void BM_orc_read_varying_options(benchmark::State& state)
       .columns(cols_to_read)
       .use_index(use_index)
       .use_np_dtypes(use_np_dtypes)
-      .timestamp_type(ts_type)
-      .decimals_as_float64(dec_as_float);
+      .timestamp_type(ts_type);
 
   auto const num_stripes              = data_size / (64 << 20);
   cudf::size_type const chunk_row_cnt = view.num_rows() / num_chunks;
```

Comment:
Can't leave a comment there, but line 190 should be modified to only vary the two remaining options:
`{0b11, 0b10, 0b01},  // `true` is default for each boolean parameter here`